src/containers/BooksForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Constants from '../common/constantModule';
+
+const BooksForm = () => (
+  <form className="bookform-section">
+    <h4>ADD NEW BOOK</h4>
+    <input type="text" />
```

Comment:
- [ ] The BookForm should contain the handleChange function and call it on the form input.